bears/java/PMDBear.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class PMDBear(LocalBear, Lint):
+    executable = 'run.sh'
```

Comment:
Unfortunately, is seems so. I will search in their documentation more, but this is what I've found to far. 
https://pmd.github.io/pmd-5.4.1/usage/running.html

"partial" solution would be renaming that sh, I think.